src/lib/api.js

```diff
@@ -976,6 +976,15 @@ const getGlobalAggregations = async (endpoint = '') => {
 }
 
 const getCatalog = async function (txnEnabled, backend, endpoint = '') {
+  const collectionsOrError = await backend.getCollections(1, COLLECTION_LIMIT)
```

Comment:
There are many cases where a response might contain an error. I don't know that we _**need**_ the `OrError` part in this name, but it doesn't really hurt either

Reply:
It's named that way because the type is Either[List[Collection],Error], so it's clear that this might be either.